packages/algob/src/types.ts

```diff
@@ -780,6 +780,8 @@ export interface ConfirmedTxInfo {
 	"confirmed-round": number;
 	"asset-index": number;
 	"application-index": number;
-	"global-state-delta": string;
-	"local-state-delta": string;
+	"global-state-delta"?: algosdk.modelsv2.EvalDeltaKeyValue;
+	"local-state-delta"?: algosdk.modelsv2.AccountStateDelta;
+	"inner-txns"?: ConfirmedTxInfo;
+	txn: algosdk.EncodedSignedTransaction;
```

Comment:
Could you explain why this one is without parenthesis?

Reply:
So { "txn" : any } will equal {txn: any}. JS "sugar syntax" I think